### PR TITLE
CI: Add Python 3.5 checks to GitHub Actions.

### PR DESCRIPTION
**Pull request by carlosperate**
As I'm updating make.py @tjguk would you be able to review this PR?

### CODE REVIEW

File: make.py [hunk 1]
```diff
@@ -170,6 +170,17 @@ def tidy():
 def black():
     """Check code with the 'black' formatter."""
     print("\nblack")
+    # Black is no available in Python 3.5, in that case let the tests continue
+    try:
+        _ = subprocess.run([TIDY, "--version"])
```
Comment (tjguk) [inline make.py:175]: No need to assign the result of `subprocess.run` if it's not being used. (Just do `subprocess.run(..)`

**Review by tjguk — APPROVED** [review]
Basically the approach looks fine. I've left one tiny comment for something which seemed unnecessary. (But which obviously works fine)

File: make.py (hunk 1 above)
Comment (carlosperate) [reply make.py:175]: Yeah, it's a bit cleaner that way, originally I was looking at the output, but ultimately we only care if it throws an exception.

**Comment by carlosperate** [comment]
The CI failures are unrelated to this PR, so I'll merge this.

File: make.py (hunk 1 above)
Comment (dybber) [reply make.py]: assigning to _ is also a bit ugly, when we use _ for localization :)

**Pull request merged**